packages/dart/npt_flutter/lib/widgets/npt_app_bar.dart

```diff
@@ -110,15 +111,15 @@ class _NptAppBarState extends State<NptAppBar> {
               ],
             ),
             actions: [
-              // IgnorePointer(
-              //   ignoring: !isDashboard,
-              //   child: AnimatedOpacity(
-              //     duration: const Duration(milliseconds: 300),
-              //     curve: Curves.easeInOut,
-              //     opacity: isDashboard ? 1 : 0,
-              //     child: const AuthorisationAppBarButton(),
-              //   ),
-              // ),
+              IgnorePointer(
+                ignoring: !isDashboard,
+                child: AnimatedOpacity(
+                  duration: const Duration(milliseconds: 300),
+                  curve: Curves.easeInOut,
+                  opacity: isDashboard ? 1 : 0,
+                  child: const AuthorisationAppBarButton(),
+                ),
+              ),
```

Comment:
backout of a previous commit on this file, to re-enable the authorisation dashboard